tests/integration/einstein/test_operations.py

```diff
@@ -222,3 +223,40 @@ def tearDown(self):
         self.db.project.delete_with_name(PROJECT)
         self.db.close()
         self.good_bmi.shutdown()
+
+
+class TestCopyImage(TestCase):
+    """
+    Creating a flatten copy of an image
+    """
+    @trace
+    def setUp(self):
+        self.db = Database()
+        self.db.project.insert(PROJECT, NETWORK)
+
+        self.good_bmi = BMI(CORRECT_HIL_USERNAME, CORRECT_HIL_PASSWORD,
+                            PROJECT)
+        self.good_bmi.import_ceph_image(EXIST_IMG_NAME)
+        img2 = uuid.uuid4()
```

Comment:
this should be `self.img2 = uuid.uuid4()`, then it will be available in `runTest` and `tearDown`. And to call it you need to write `self.img2`